modules/@apostrophecms/modal/ui/apos/components/TheAposModals.vue

```diff
@@ -5,58 +5,68 @@
       :is="modal.componentName"
       v-for="modal in store.stack"
       :key="modal.id"
-      :modal-id="modal.id"
+      :modal-data="modal"
       @modal-result="store.setModalResult(modal.id, $event)"
     />
   </div>
 </template>
 
```

Comment:
Updated the component to use `setup () {}` method instead of the `script setup` because it's a root component and we don't have the name of the component with the script setup.